### DISCUSSION TITLE
The bottom line of GO help window isn't visible on Linux

### DISCUSSION THREAD

**Discussion by William-Purvis** [GrandOrgue/grandorgue#1472] [Developers]
I have found it impossible to read the last line of the help window. Scrolling down to the bottom, leaves the lest line hidden by the bottom margin (which usually  says 'Done'). Maybe all the help page should end with a couple of blank lines?

**Comment by ClarkeBrunt** [comment]
Seeing as no other response to this yet, here's something perhaps not very helpful from me. I just tried looking at the help. Indeed there is some kind of status bar at the bottom of the window which reads 'Done', but the few last lines of pages which I've looked at seem to be visible.

I think people are going to want to know which GO version, which Linux distro, whether Wayland or X as display server, and a specific help page which shows the problem.

I'm on GO v3.10.1-1 with Ubuntu Studio 22.04.2 LTS (which uses X, not Wayland).

**Reply by William-Purvis** [reply]
OK, I'm on Ubuntu 22.04.2 LTS, using X not Wayland. I'm running GO 3.10.-0.local (compiled locally but with no modifications).
Here's a screenshot of the help window. It applies to any help page that is longer than the window, i.e. you need to scroll down.
In This case it's the Program Settings page.

![Screenshot at 2023-04-19 14-24-17](https://user-images.githubusercontent.com/32897843/233090059-23d25891-dd2e-4510-b639-55f06cf2e4f1.png)

I have just downloaded and built the latest version 3.11.1-0 on Devuan Chimaera 4.0, again using X, and that shows the same effect, but slightly worse! (Metronome only shows the top of the heading.)

**Reply by William-Purvis** [reply]
Sorry, version in first line above should read GO 3.10.0-0

**Reply by ClarkeBrunt** [reply]
I just looked at that particular help page (as in the screenshot) and it appears fine with my configuration (as above) - I can see right to the bottom of the text. So who knows? I imagine a different version of some library or other maybe.

**Comment by midimusic** [comment]
I have just checked GO v3.2.0-1 on windows and there is no problem.

However GO uses wxWidgets for handling all its graphics. The displayed results may alter depending on which version of wxWidgets was used when compiling. Also programming exact shapes and sizes in wxWidgets is virtually impossible because it calculates these itself from sometimes vague information provided by the programmer. There are variations between operating systems and graphics renderers. These can only be discovered by trial and error and even when discovered not much can be done about the problem.

Also I have found that wxWidgets contains many peculiarities and is particularly unreliable on the most recent graphics systems especially in the area of scaling.

csw900
